query-connector/src/app/database-service.ts

```diff
@@ -377,8 +376,8 @@ function generateValuesetConceptJoinSqlPromises(vs: ValueSet) {
     RETURNING valueset_id, concept_id;
     `;
     const conceptInsertPromise = dbClient.query(insertJoinSql, [
-      `${valueSetUniqueId}_${conceptUniqueId}`,
-      valueSetUniqueId,
+      `${vs.valueSetId}_${conceptUniqueId}`,
+      vs.valueSetId,
```

Comment:
nice catch!

Reply:
This thing gave me no end of problems when debugging. Not even the regular insert was working because the valueset got inserted regularly but then the foreign keys for `valueset_to_concept` all had the version number appended a second time so the DB was imploding 😭